theseus/embodied/measurements/between.py

```diff
@@ -37,8 +37,11 @@ def error(self) -> torch.Tensor:
 
     def jacobians(self) -> Tuple[List[torch.Tensor], torch.Tensor]:
         Jlist: List[torch.Tensor] = []
-        var_diff = between(self.v0, self.v1, jacobians=Jlist)
-        error = self.measurement.local(var_diff)
+        var_diff = between(self.v0, self.v1)
+        log_jac: List[torch.Tensor] = []
+        error = self.measurement.inverse().compose(var_diff).log_map(jacobians=log_jac)
```

Comment:
Do we need the product of jacobians of all the operations (dcompose, dbetween, etc) involved here to get the final d(error)/d(v0) and d(error)/d(v1)?

Reply:
Not necessarily. I think this should be the most efficient way to calculate the derivatives.